src/CallingExtensions.js

```diff
@@ -57,44 +35,70 @@ class CallingExtensions {
     });
   }
 
-  initialized(userData) {
+  /**
+   *
```

Comment:
Could you please use the same descriptions as our [developer docs](https://developers.hubspot.com/docs/api/crm/extensions/calling-sdk#events) where possible in this file?